### PR TITLE
Changes to docs to account for RNA-seq sample compendia

### LINKED ISSUE #109: Add brief information on quantpendia
Describe when we might not be able to run tximport. These remain available as quant files. Note that these can be downloaded via the quantpendium. Also link to the API docs in case someone wants to use the API.

### PR DESCRIPTION
**Pull request by jaclyn-taroni**
Closes #109 and #114

* Adds a section about RNA-seq sample compendia and reorganizes the species compendia section accordingly
* Updates the getting started section to read as normalized compendia

### CODE REVIEW

File: docs/main_text.md [hunk 1]
```diff
@@ -432,12 +437,28 @@ We then quantile normalize all samples as described above.
...
+### Download Folder
+
 Users will receive a zipped folder with a gene expression matrix aggregated by species, along with associated metadata.
 Below is the detailed folder structure:
 
 ![docs-downloads-species-compendia](https://user-images.githubusercontent.com/15315514/65180873-ddbb4f80-da2b-11e9-97e9-127c68106182.png)
 
+## RNA-Seq Sample Compendia
+
+refine.bio RNA-seq sample compendia are comprised of the Salmon output for the collection of RNA-seq samples from an organism that can be processed with refine.bio.
```
Comment (cgreene) [inline docs/main_text.md]:
```suggestion
refine.bio RNA-seq sample compendia are comprised of the Salmon output for the collection of RNA-seq samples from an organism that we have processed with refine.bio.
```

File: docs/main_text.md [hunk 2]
```diff
@@ -432,12 +437,28 @@ We then quantile normalize all samples as described above.
...
+Each individual sample has its own `quant.sf` file; the samples have not been aggregated and normalized.
+RNA-seq sample compendia are designed to allow users that are comfortable handling these files to generate output that is most useful for their downstream applications. 
+Please see the [Salmon documentation on the `quant.sf` output format](https://salmon.readthedocs.io/en/latest/file_formats.html#quantification-file) for more information.
+
+### Download Folder
+
+Users will receive a zipped folder with individual `quant.sf` files for each sample that we were able to process with Salmon, grouped into folders based on the experiment those samples come from, along with any associated metadata in refine.bio.
+Please note that our RNA-seq sample metadata is limited at this time and in some cases, we could not successfully run Salmon on every sample within an experiment (e.g., the sequence files were malformed).
```
Comment (cgreene) [inline docs/main_text.md]:
```suggestion
Please note that our RNA-seq sample metadata is limited at this time and in some cases, we could not successfully run Salmon on every sample within an experiment (e.g., our processing infrastructure encountered an error with the sample, the sequencing files were malformed).
```

File: docs/main_text.md [hunk 3]
```diff
@@ -408,8 +408,13 @@ Currently, we only support skipping quantile normalization for RNA-seq experimen
 
 We periodically release compendia comprised of all the samples from a species that we were able to process.
 We refer to these as **species compendia**.
+We offer two kinds of species compendia: [normalized compendia](#normalized-compendia) and [RNA-seq sample compendia](#rna-seq-sample-compendia).
```
Comment (dvenprasad) [inline docs/main_text.md]: we refer to them as refine.bio compendia now?

**Comment by jaclyn-taroni** [comment]
Seems easier to change here. I will also change the section header.

**Comment by dvenprasad** [comment]
Updated normalized compendia pipeline

docs-notmalized-compendia

**Pull request merged**